kythe/go/platform/tools/extraction/repotester.go

```diff
@@ -16,7 +16,7 @@
 
 // Binary repotester tests the high level validity of repo extraction. It tries
 // to run similar logic to sibling binary extractrepo on a specified target
-// repositories, and reports results.
+// repositories, and reports results in csv format.
```

Comment:
If we really want to use CSV, we should probably use https://godoc.org/encoding/csv so that things get quoted correctly if there are internal punctuation/whitespace.